npm-shrinkwrap.json

```diff
@@ -4988,6 +4988,19 @@
             "supports-color": "^5.3.0"
           }
         },
+        "cli-table": {
```

Comment:
can you please revert those two (`cli-table` and `colors`), I'm pretty sure we do not need them